sample-app/src/components/VisualAutocomplete/SampleVisualSearchBar.tsx

```diff
@@ -12,6 +13,12 @@ export default function SampleVisualSearchBar() {
       screenReaderInstructionsId='SearchBar__srInstructions'
       headlessId='visual-autocomplete'
       entityPreviewsDebouncingTime={150}
+      handleVerticalLinks={verticalKeys => {
```

Comment:
I was thinking handleVerticalLink would be responsible for changing the page to the correct vertical, that way DropdownSection doesn't have to know about react router

Reply:
we would need some sort of mapping still to get the right label to display in the dropdown beside navigation

Reply:
Ohh I missed that. Maybe this prop can just be for transforming a vertical key to the right label? feel like this function taking in an array of keys is a little arbitrary
```ts
verticalKeyToLabel={{verticalKey => { 
  const label = universalResultsConfig[verticalKey]?.label ?? verticalKey;
  return { label, verticalKey }
}}
```

Reply:
sounds good, updated!